src/edu/csus/ecs/pc2/clics/API202003/JSONTool.java

```diff
@@ -119,40 +116,26 @@ private String getAPIURL() {
         return url;
     }
 
-    private void logWarn(String string, Exception e) {
-
-        System.err.println(string);
-        e.printStackTrace(System.err);
-
-        Log log = controller.getLog();
-        log.log(Level.WARNING, string, e);
-    }
-
+    @Override
     public ObjectNode convertToJSON(Group group) {
         ObjectNode element = mapper.createObjectNode();
-        element.put("id", getGroupId(group));
+        element.put("id", IJSONTool.getGroupId(group));
```

Comment:
I've noted elsewhere my question/concern about using static methods in this interface.

Reply:
Same reply.